packages/frontend-shared/src/locales/en-US.json

```diff
@@ -218,7 +218,8 @@
         "content": "Record a run to see your test results in Cypress Cloud. You can then optimize your test suite, debug failing and flaky tests, and integrate with your favorite tools."
       }
     },
-    "runAllSpecs": "Run {n} spec | Run {n} specs"
+    "runSelectedSpecs": "Run {n} spec | Run {n} specs",
+    "runAllSpecs": "Run all specs"
```

Comment:
I'd suggest we do like 9.x, and when there's a filter active, make the wording be "run filtered specs" for clarity about what to expect.

Reply:
I did this here https://github.com/cypress-io/cypress/pull/24846/commits/8e12cbbfc1a70eafa619aae6dff967be8bbb3136